docs/docs/custom-schemas.md

```
@@ -27,9 +27,14 @@ const (
 	Dev  Env = "env"
 )
 
+type ActiveInactive int
+const (
+	Active int = 1
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)